apps/cli/src/constants.ts

```diff
@@ -0,0 +1,33 @@
+import { join } from "node:path";
+import envPaths from "env-paths";
+
+const APP_CONFIG = envPaths("webstudio").config;
+const GLOBAL_CONFIG_FILE_NAME = "webstudio-config.json";
+export const GLOBAL_CONFIG_FILE = join(APP_CONFIG, GLOBAL_CONFIG_FILE_NAME);
+
+export const CLI_ARGS_OPTIONS = {
+  debug: {
+    type: <const>"boolean",
+    short: "d",
+    default: false,
+  },
+  version: {
+    type: <const>"boolean",
+    short: "v",
+  },
+  help: {
+    type: <const>"boolean",
+    short: "h",
+  },
+};
+
+export const HELP = `Usage:
```

Comment:
Code looks a bit fragmented with all these constants, utils, types. Looks like .html,  .css and .js. Let's group logic related to parsing/help like args.ts with showHelp and parseCommands